throw new ${exception_name}(${data_expression}\
.if (cause != "")
${cause}\
.end if
);
